### PR TITLE
OCPBUGS-33687: Updating prune memory limits

### PR DESCRIPTION
**Pull request by MiguelCarpio**
OpenStack NFV cleanup failed. OutOfMemoryKilled issue. The build openstack-installer-amd64 failed.

The periodic-ci-shiftstack-prune-main-cleanup-openstack-nfv runs four times per day, but we frequently get an OutOfMemoryKilled error once or twice daily.

The typical error log is:

```
Build shiftstack-prune-image-amd64 previously failed from an infrastructure error (OutOfMemoryKilled), retrying... 
Created build "shiftstack-prune-image-amd64" "
```

### CODE REVIEW

**Comment by MaysaMacedo** [comment]
/pj-rehearse periodic-ci-shiftstack-prune-main-cleanup-openstack-nfv

File: ci-operator/config/shiftstack/prune/shiftstack-prune-main.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ images:
 resources:
   '*':
     limits:
-      memory: 1Gi
+      memory: 4Gi
```
Comment (MaysaMacedo) [inline ci-operator/config/shiftstack/prune/shiftstack-prune-main.yaml:28]: 4 might be too much. Have you tried with 2?

Comment (MiguelCarpio) [reply ci-operator/config/shiftstack/prune/shiftstack-prune-main.yaml:28]: No. Let's try with 2Gi first.

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-prune-main-cleanup-openstack-nfv

File: ci-operator/config/shiftstack/prune/shiftstack-prune-main.yaml (hunk 1 above)
Comment (MiguelCarpio) [reply ci-operator/config/shiftstack/prune/shiftstack-prune-main.yaml]: Done

**Pull request merged**